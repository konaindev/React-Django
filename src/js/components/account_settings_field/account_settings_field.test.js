import React from "react";
import renderer from "react-test-renderer";

import Input from "../input";
import AccountSettingsField from "./index";

describe("AccountSettingsField", () => {
  it("renders AccountSettingsField correctly", () => {
    const tree = renderer
      .create(
        <AccountSettingsField label="Test Field" errorName="field">
          <Input className="account-settings-field__input" theme="gray" />
        </AccountSettingsField>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
