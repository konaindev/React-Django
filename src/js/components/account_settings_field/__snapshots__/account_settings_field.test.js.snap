// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AccountSettingsField renders AccountSettingsField correctly 1`] = `
<div
  className="account-settings-field"
  errorName="field"
>
  <div
    className="account-settings-field__label"
  >
    Test Field
  </div>
  <input
    className="input account-settings-field__input input--gray"
    onChange={[Function]}
    type="text"
  />
</div>
`;
